.github/workflows/app-create-verify.yml

```diff
@@ -0,0 +1,132 @@
+name: Verify Project
+
+#######
+# Verify creating a project for a target framework on a particular OS.
+#######
+
+on:
+  workflow_call:
+    inputs:
+      python-version:
+        description: "Python version to use; defaults to latest Python release."
+        default: "3.X"
+        type: string
+      runner-os:
+        description: "The OS to use to build the App; must be a fully qualified GitHub runner OS, e.g. ubuntu-latest."
+        required: true
+        type: string
+      framework:
+        description: "Framework to use to build the App, e.g. toga."
+        required: true
+        type: string
+      briefcase-template-source:
+        description: "Path to use for --template for `briefcase new` to create app."
+        default: ""
+        type: string
+      briefcase-template-branch:
+        description: "Git branch to use for --template-branch for `briefcase new` to create app."
+        default: ""
+        type: string
+
+defaults:
+  run:
+    shell: bash
+
+env:
+  FORCE_COLOR: "1"
+
+jobs:
+  verify-app:
+    name: Verify App Create
+    runs-on: ${{ inputs.runner-os }}
+    steps:
+
+    - name: Checkout ${{ github.repository }}
+      uses: actions/checkout@v4.1.1
+      with:
+        repository: ${{ github.repository }}
+        fetch-depth: 0
+
+    - name: Checkout beeware/.github
+      uses: actions/checkout@v4.1.1
+      with:
+        repository: rmartin16/.github-beeware  # !!!! TODO:PR: REPLACE ME WITH beeware/.github !!!!
+        ref: create-new-project  # !!!! TODO:PR: REMOVE ME !!!!
+        path: beeware-.github
+
+    - name: Checkout beeware/briefcase-template
+      uses: actions/checkout@v4.1.1
+      with:
+        repository: rmartin16/briefcase-template  # !!!!! TODO:PR: REPLACE ME WITH beeware/briefcase-template !!!!!
+        ref: gui-plugin-support  # !!!!! TODO:PR REMOVE ME !!!!!
+        path: briefcase-template
```

Comment:
It isn't clear why I added this....but it definitely isn't necessary now. It should be removed when the updates are made prior to merging.

Reply:
Ohh nvm, i see now. This is needed to install the requirements for linting the project. So, it should stick around.